tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/matterdashboard.controller.js

```diff
@@ -1998,7 +1998,7 @@
             // Function to configure time stamp
             vm.overwriteConfiguration = function (fileName) {
                 // Update the content as per the logic.
-                var selectedOverwriteConfiguration = vm.uploadMessages.overwrite_Config_Property.trim().toLocaleUpperCase(),
+                var selectedOverwriteConfiguration = vm.globalSettings.overwriteDupliacteFileNameWithDateTimeFor.trim().toLocaleUpperCase(),

```

Comment:
Instead of reading the value from overwrite_Config_Property, now reading from overwriteDupliacteFileNameWithDateTimeFor